### ISSUE TITLE
Form Template's "Output Theme" setting is not respected in the staging/production environment

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#649]
**Description**

I've run into a strange issue with the default Form Template's Output Theme setting, and I'm at a bit of a loss trying to figure out what the cause of the issue is. 

In my development environment, under _Formie > Settings > Form Templates > Default_, I've turned the "Output Theme" option off but have left the "Output CSS" option on. This works as expected and I see this HTML code added to my document on the frontend, right before my closing `</head>` tag:
```html
<link href="http://www.myproject.test/cpresources/c7bfab6f/formie-base.css?v=1637577811" rel="stylesheet">
```

However, in my staging environment, this is what I get on the frontend, right before the closing `</head>` tag:
```html
<link href="https://test.myproject.com/cpresources/c7bfab6f/formie-theme.css?v=1637585047" rel="stylesheet">
```

My dev environment is synced to the staging environment using Craft's standard Project Config system. It appears that the config settings _are_ being synced correctly, because the Form Templates table in the staging environment contains all the correct settings.

**Dev Environment**
```
mysql> select * from formie_formtemplates\G;
*************************** 1. row ***************************
                id: 1
              name: Default
            handle: default
          template: 
useCustomTemplates: 0
   outputCssLayout: 1
    outputCssTheme: 0
      outputJsBase: 1
     outputJsTheme: 1
 outputCssLocation: page-header
  outputJsLocation: page-footer
         sortOrder: 99
     fieldLayoutId: NULL
       dateDeleted: NULL
       dateCreated: 2021-08-09 09:18:00
       dateUpdated: 2021-11-22 10:57:27
               uid: a69c6bdb-50db-4e87-b6ae-e1c321670a0c
1 row in set (0.00 sec)
```

**Staging Environment**
```
mysql> select * from formie_formtemplates\G;
*************************** 1. row ***************************
                id: 1
              name: Default
            handle: default
          template: 
useCustomTemplates: 0
   outputCssLayout: 1
    outputCssTheme: 0
      outputJsBase: 1
     outputJsTheme: 1
 outputCssLocation: page-header
  outputJsLocation: page-footer
         sortOrder: 99
     fieldLayoutId: NULL
       dateDeleted: NULL
       dateCreated: 2021-08-09 12:07:09
       dateUpdated: 2021-08-09 12:07:09
               uid: a69c6bdb-50db-4e87-b6ae-e1c321670a0c
1 row in set (0.01 sec)
```

The event logs in my dev environment mention the `FormTemplate` class:
| Time | Name | Class | Sender | Static |
| --- | --- | --- | --- | --- |
14:24:31.248 | defineBehaviors | craft\events\DefineBehaviorsEvent | verbb\formie\models\FormTemplate | No
14:24:31.248 | init | yii\base\Event | verbb\formie\models\FormTemplate | No

...but the event logs in the staging environment make no mention of this class at all.

Lastly, in my dev environment database logs, I see that a record is being fetched from the `formie_formtemplates` table:
```mysql
SELECT `id`, `name`, `handle`, `template`, `useCustomTemplates`, `outputCssLayout`, `outputCssTheme`, `outputJsBase`, `outputJsTheme`, `outputCssLocation`, `outputJsLocation`, `sortOrder`, `fieldLayoutId`, `dateDeleted`, `uid`
FROM `formie_formtemplates`
WHERE `dateDeleted` IS NULL
ORDER BY `sortOrder`
```
...but this MySQL query does *not* appear in the database logs in the staging environment.

I'm at a loss trying to figure out why this is happening. I use Formie on a number of website, but I've only run into this problem on one of them so far. The only conclusion I've been able to come to is that the correct settings exist in the staging environment, but the code that is supposed to fetch the form template data is never run for some reason. Do you have any idea why this might be happening?

Thanks!

### Form settings
- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.25
- Craft version: 3.7.21
- Multi-site: No

**Comment by engram-design** [comment]
Quick question - have you assigned your form the Default form template you've created? You can't actually edit the "real default" form template from Formie, instead you have to create a new one, and use that as your default template (which only affects newly created forms). Any form you've already created won't have their form template updated.

Just thought to check that before going any further?

**Comment by mtnorthrop** [comment]
@engram-design , thank you very much for your response.

I checked the form's "Form Templates" option under the Appearance tab and I saw that it was set to "Default Formie Template". I changed this to "Default" and everything then worked as expected.

I then tested creating a two new forms in the staging environment, once using a blank stencil and once using the Contact Form stencil, and both were created with the "Default" template selected, instead of "Default Formie Template".

I'm not sure how the original form ended up having "Default Formie Template" chosen, but everything is now working exactly as expected.

Thanks again for your time!
